archetypes/helidon/src/main/archetype/common/extra.xml

```diff
@@ -101,6 +101,11 @@ java -cp "target/classes:target/libs/*" {{package}}.WebClientMain PORT
                                     <value key="groupId">io.helidon.microprofile.metrics</value>
                                     <value key="artifactId">helidon-microprofile-metrics</value>
                                 </map>
+                                <map>
+                                    <value key="groupId">io.helidon.metrics</value>
+                                    <value key="artifactId">helidon-metrics</value>
+                                    <value key="scope">compile</value>
```

Comment:
IIUC this is used in the MP case. `helidon-metrics` includes `helidon-metrics-api` which we would normally not need at build-time for MP. It's certainly OK for an MP app to use the Helidon neutral metrics API but it's a bit unusual.

Can this be `runtime` scope instead or do we need it to be `compile`?

Reply:
Sure, done !